.github/workflows/docker.build.yml

```diff
@@ -4,14 +4,19 @@ on:
   push:
     branches:
       - 'main'
-    tags:
-      - '*'
   pull_request:
+  workflow_dispatch:
```

Comment:
so when would you trigger this workflow dispatching on a branch that isn't already a PR?